101-identitygovernance-entitlementmanagement-extensibility-sample-logicapp/azuredeploy.parameters.json

```diff
@@ -1,8 +1,12 @@
 {
-   "name":{
-      "value":"ELMCustomAction"
-   },
-   "location":{
-      "value":"westcentralus"
-   }
+	"$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentParameters.json#",
+	"contentVersion": "1.0.0.0",
+	"parameters": {
+		"name": {
+			"value": "ELM-custom-action"
+		},
+		"catalogId": {
+			"value": ""
```

Comment:
if this is not required - remove the param from here and add a defaultValue of emptyString in the template - also add a note to the description saying it's optional.

Reply:
This is required and so left it here but updated with a sample Id which is a Guid

Reply:
What does the user need to provide?  A specific GUID?  If so, state what that is in the readme (or how the user gets the GUID).  Or any GUID?  If so, should it be idempotent?  If so, use the guid() function in the defaultValue of the param, if not use the newGuid() function.

Reply:
this is not any guid from guid from ELM , so updated the readme to have this explained.